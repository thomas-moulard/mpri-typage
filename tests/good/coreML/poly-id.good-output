Fatal error: exception Assert_failure("infer.ml", 72, 12)
