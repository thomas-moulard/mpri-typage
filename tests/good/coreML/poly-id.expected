id = fun x.x

id : forall 'a.('a -> 'a)
